sds_in_a_box/SDSCode/api.py

```diff
@@ -0,0 +1,68 @@
+import json
+import os
+import requests
+import time
+from jose import jwk, jwt
+from jose.utils import base64url_decode
+
+def _verify_cognito_token(token):
+    region = 'us-west-2'
+    userpool_id = os.environ["COGNITO_USERPOOL_ID"]
+    app_client_id = os.environ["COGNITO_APP_ID"]
+    keys_url = f'https://cognito-idp.{region}.amazonaws.com/{userpool_id}/.well-known/jwks.json'
```

Comment:
What are we storing there? a pub/private ssh key or something like that?

Reply:
Yeah cognito automatically sets up these "JSON Web Key Sets" that have the public keys that you use to verify someone's JSON Web Token.  It comes automatically with a new cognito user group.  For example, here is the public key for the latest version of the code I have set up in the development environment: https://cognito-idp.us-west-2.amazonaws.com/us-west-2_rHAHpNHQi/.well-known/jwks.json

Reply:
I see.